packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -268,14 +269,15 @@ const VirtualListBaseFactory = (type) => {
 				// remove a function for preventing native scrolling by Spotlight
 				if (containerNode && containerNode.removeEventListener) {
 					containerNode.removeEventListener('scroll', this.preventScroll);
-					containerNode.removeEventListener('keydown', this.onKeyDown);
 				}
 			}
 
 			if (containerNode && containerNode.removeEventListener) {
 				containerNode.removeEventListener('keydown', this.onKeyDown);
 			}
 
+			this.resumeSpotlight();
```

Comment:
If `isItemDisabled` prop is not defined in `VirtualList`, `this.virtualListPause.resume();` in `resumeSpotlight()` generates an error when `this.resumeSpotlight()` is called because this.virtualListPause is `null`.